recipes-core/images/iot2050-image-swu-example.bb

```diff
@@ -25,7 +25,9 @@ SWU_ROOTFS_TYPE:secureboot = "verity"
 WIC_IMAGER_INSTALL += "efibootguard"
 # watchdog is managed by U-Boot - disable
 WDOG_TIMEOUT = "0"
-WICVARS += "WDOG_TIMEOUT KERNEL_IMAGE INITRD_IMAGE DTB_FILES"
+WICVARS += "WDOG_TIMEOUT KERNEL_IMAGE INITRD_DEPLOY_FILE DTB_FILES"
+
+INITRD_DEPLOY_FILE = "${INITRAMFS_RECIPE}-${DISTRO}-${MACHINE}.initrd.img"
```

Comment:
Should be folded into previous commit to avoid git bisection breakage.

Reply:
Acked and done.